### PR TITLE
[hotfix][flink-avro] Use local actual schema variable

### PR DESCRIPTION
**Pull request by mans2singh**
## What is the purpose of the change

* The variable actualSchema is declared but not used.

## Brief change log

* Removed unused variable.

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not applicable

### CODE REVIEW

**Comment by mans2singh** [comment]
@leonardBang @wuchong - Can you please review this PR and let me know your feedback ?  Thanks

File: flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/AvroRowDeserializationSchema.java [hunk 1]
```diff
@@ -243,7 +243,6 @@ private Object convertAvroType(Schema schema, TypeInformation<?> info, Object ob
             case UNION:
                 final List<Schema> types = schema.getTypes();
                 final int size = types.size();
-                final Schema actualSchema;
```
Comment (leonardBang) [inline flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/AvroRowDeserializationSchema.java:246]: How about giving a meaningful `actualSchema` ?
```
                final Schema actualSchema;
                if (size == 2 && types.get(0).getType() == Schema.Type.NULL) {
                    actualSchema = types.get(1);
                } else if (size == 2 && types.get(1).getType() == Schema.Type.NULL) {
                    actualSchema = types.get(0);
                } else if (size == 1) {
                    actualSchema = types.get(0);
                } else {
                    // generic type
                    return object;
                }
                return convertAvroType(actualSchema, info, object);
```

**Review by leonardBang — APPROVED** [review]
LGTM, wait the CI passed

**Comment by mans2singh** [comment]
@leonardBang  @wuchong  - Thanks for your advice.   Please let me know if there is anything else required.  Thanks.

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**